src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -2484,7 +2491,96 @@ paths:
           description: Unexpected error
           schema:
             $ref: '#/definitions/Error'
+  '/nodes/{nodeId}/calculateSize':
+    post:
+      tags:
+        - nodes
+      summary: Calculate a folder size
+      description: |
+        
+        This may be executed against a node specified by **nodeId** and this nodeId must be a valid node ID.
+        Furthermore, this POST endpoint will be executed **asynchronously** with a `202` HTTP response signifying that
+        the request has been accepted successfully.
+        
+        The response body will include the ```nodeId``` of the pending action, which can be used in a
+        **GET/calculateSize** endpoint to check if the action's status has been completed, at which point the result will be returned, comprising ```the size of the node in bytes```.
+        
+        Essentially, ```maxItems``` ***parameter*** is utilized for pagination purposes.
+        
+        By default, **results are limited to the first 100.**
```

Comment:
I read the documentation above and here, and I still don't know what will happen if I set `maxItems`. Will I get the size of the first `maxItems` in the folder, or will it be an estimate of the total size based on the first `maxItems`?

Reply:
To efficiently retrieve and consolidate size of large, nested data, we are using **maxItems** parameter to divide the data into segments.